iree/compiler/Dialect/LinalgExt/IR/test/roundtrip.mlir

```diff
@@ -305,17 +305,61 @@ func @fft_tensor(%arg0: tensor<1024xf32>, %arg1: tensor<1024xf32>)
 // -----
 
 func @fft_memref(%arg0: memref<1024xf32>, %arg1: memref<1024xf32>) {
-  %cst0 = constant 0 : index
+  %cst1 = constant 1 : index
   linalg_ext.fft
-    ins(%cst0: index)
+    ins(%cst1: index)
     outs(%arg0, %arg1: memref<1024xf32>, memref<1024xf32>)
   return
 }
 // CHECK-LABEL: func @fft_memref(
 //  CHECK-SAME:   %[[REAL:[a-zA-Z0-9_]+]]
 //  CHECK-SAME:   %[[IMAG:[a-zA-Z0-9_]+]]
-//       CHECK:   %[[CST:.+]] = constant 0 : index
+//       CHECK:   %[[CST:.+]] = constant 1 : index
 //       CHECK:   linalg_ext.fft
 //  CHECK-SAME:     ins(%[[CST]] : index)
 //  CHECK-SAME:    outs(%[[REAL]], %[[IMAG]] : memref<1024xf32>, memref<1024xf32>)
 //       CHECK:   return
+
+// -----
+
+func @fft_tensor_coef(%arg0: tensor<1024xf32>, %arg1: tensor<1024xf32>,
+    %arg2: tensor<1xf32>, %arg3: tensor<1xf32>) -> (tensor<1024xf32>, tensor<1024xf32>) {
+  %cst1 = constant 1 : index
+  %0:2 = linalg_ext.fft
+    ins(%cst1, %arg2, %arg3: index, tensor<1xf32>, tensor<1xf32>)
```

Comment:
Is the size of this tensor same as the value of the stage? Maybe worth adding that to the verifier and also add a test for the non-1 stage?

Reply:
No, it should be `2^(stage-1)`. I will add non-1 stage cases.